packages/office-ui-fabric-react/src/utilities/color/_rgbaOrHexString.ts

```diff
@@ -0,0 +1,10 @@
+import { MAX_COLOR_ALPHA } from './consts';
+
+/**
+ * Get a CSS color string from some color components.
+ * If `a` is specified and not 100, returns an `rgba()` string.
+ * Otherwise returns `hex` prefixed with #.
+ */
+export function _rgbaOrHexString(r: number, g: number, b: number, a: number | undefined, hex: string): string {
```

Comment:
I really don't like having this internal function in a separate file. Is there anything we can group it with? (Also, you should add `@internal` to the doc comment.)

Reply:
Unfortunately it's used by multiple files... I did not export this out of colors.ts or out of index.ts.